views/includes/sign-out-user-banner.html

```diff
@@ -1,8 +1,17 @@
 {% if userEmail %}
+
+  {% set signOutPath = 'sign-out?page=' + templateName %}
+
+  {% if journey === 'remove' %}
+    {% set url = url if url else OE_CONFIGS.UPDATE_AN_OVERSEAS_ENTITY_URL %}
```

Comment:
This just looks a bit confusing - setting 'update' as the url if we're on a remove journey. I assume this is handling reuse of the update pages in the remove journey, but worth a comment I think, explaining this for future readers of this code (or me, in a week's time when I've forgotten most things apart from my name).